src/ray/raylet/lineage_cache.cc

```diff
@@ -0,0 +1,39 @@
+#ifndef LINEAGE_CACHE_CC
+#define LINEAGE_CACHE_CC
+
+#include "lineage_cache.h"
+
+#include <stdexcept>
+
+namespace ray {
+
+LineageCache::LineageCache() {}
+
+ray::Status LineageCache::AddTask(const Task &task) {
+  throw std::runtime_error("method not implemented");
+  return ray::Status::OK();
+}
+
+ray::Status LineageCache::AddTask(const Task &task,
+                                  const Lineage &uncommitted_lineage) {
+  throw std::runtime_error("method not implemented");
+  return ray::Status::OK();
+}
+
+ray::Status LineageCache::AddObjectLocation(const ObjectID &object_id) {
+  throw std::runtime_error("method not implemented");
+  return ray::Status::OK();
+}
+
+Lineage &LineageCache::GetUncommittedLineage(const ObjectID &object_id) {
+  throw std::runtime_error("method not implemented");
+}
+
+Status LineageCache::Flush() {
```

Comment:
Let's be consistent.  I see `Status` and `ray::Status`.

Reply:
yeah, good point, I believe that here we may have some ambiguity between `arrow::Status` and `ray::Status`. We should disambiguate in any case.